documentation/source/building.rst

```diff
@@ -242,7 +242,7 @@ PyGPI
     The entry module must have the following additional functions defined.
     These additional requirements on the entry module may be relaxed over time.
 
-    * ``_sim_event(level: int) -> None``
+    * ``_sim_event(message: str) -> None``
```

Comment:
Might consider adding a `.. versionchanged::1.8` note here, to start noting changes over time.